manifest.md

```diff
@@ -42,13 +42,19 @@ Unlike the [image index](image-index.md), which contains information about a set
 
         If the manifest uses a different media type than the above, it MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
 
+        See [#GuidelinesForArtifactUsage] for other uses of the `config.mediaType` field.
+
 - **`layers`** *array of objects*
 
     Each item in the array MUST be a [descriptor](descriptor.md).
-    The array MUST have the base layer at index 0.
-    Subsequent layers MUST then follow in stack order (i.e. from `layers[0]` to `layers[len(layers)-1]`).
-    The final filesystem layout MUST match the result of [applying](layer.md#applying-changesets) the layers to an empty directory.
-    The [ownership, mode, and other attributes](layer.md#file-attributes) of the initial empty directory are unspecified.
+    The array MUST have at least one entry.
+
+    When the `config.mediaType` is set to `application/vnd.oci.image.config.v1+json`, the following additional restrictions apply:
```

Comment:
Do these restrictions also apply to legacy data types?

Do they not apply to artifact types?

I'm confused by this language in light of #1025, is the goal to describe what registries have already implemented, or is the goal of the specification to advance the industry by defining new benchmarks for registries to meet to reach OCI conformance?

Reply:
From the fact finding, the content of the layer wasn't important to the registry, other than being non-zero and having at least one layer. This moves the verbiage around the structure of the layers, which is a client side concern, to only apply for the image.config types of images. That allows artifacts to structure their layers specific to their artifact type, any content in any order that the artifact defines in their own spec outside of OCI.